src/components/Form.js

```diff
@@ -1,12 +1,38 @@
-const Form = () => (
-  <div className="form">
-    <form>
-      <h3>Add a Book</h3>
-      <input type="text" name="title" placeholder="title" />
-      <input type="text" name="author" placeholder="author" />
-      <button type="submit">Add Book</button>
-    </form>
-  </div>
-);
+import uniqid from 'uniqid';
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { booksActions, addBook } from '../redux/books/bookslice';
+
+function Form() {
+  const dispatch = useDispatch();
+  const [title, setTitle] = useState('');
+  const [author, setAuthor] = useState('');
+
+  const addBookHandler = (e) => {
+    e.preventDefault();
+    const books = {
+      item_id: uniqid(),
+      title,
+      author,
+      category: 'Fiction',
+    };
+
+    dispatch(booksActions.addBook(books));
+    dispatch(addBook(books));
+    setTitle('');
+    setAuthor('');
+  };
```

Comment:
- [ ] At the moment, when a book is added it displays on the screen but when the page is refreshed, it disappears. Please kindly fix this.